rero_ils/modules/vendors/api.py

```diff
@@ -55,17 +57,24 @@ class Vendor(IlsRecord):
 
     def get_number_of_acq_orders(self):
         """Get number of acq orders."""
-        from ..acq_orders.api import AcqOrdersSearch
-        results = AcqOrdersSearch().filter(
+        return AcqOrdersSearch().filter(
+            'term', vendor__pid=self.pid).source().count()
+
+    def get_number_of_acq_invoices(self):
+        """Get number of acq invoices."""
```

Comment:
Can you replace acq in the docstring with a real word? Thanks.

Reply:
done